src/frontend-pwa/src/views/Report/Report.tsx

```diff
@@ -48,84 +83,70 @@ export function Report() {
     };
 
     try {
-      // Send JSON object to the API endpoint
-      const response = await fetch('https://wf-api-ec1236-dev.apps.silver.devops.gov.bc.ca/api/report', {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(formData),
-      });
-
-      if (response.ok) {
-        // Reset form fields
-        setEventType('');
-        setDetails('');
-        setPhoneNumber('');
-      } else {
-        throw new Error('Form data submission failed');
-      }
+      await axios.post(`${constants.BACKEND_URL}/api/report`, formData);
+      setErrorMessage('');
+      setEventType('');
+      setDetails('');
+      setPhoneNumber('');
     } catch (error) {
-      throw new Error('fetch failed');
+      throw Error('Error: Form was not submitted');
```

Comment:
```suggestion
      console.error(error);
```

Reply:
This is also a good opportunity to inform the user that something went wrong as well. I triggered that error on purpose, but as a user I was not notified. I had to look in the console to know anything happened.

Reply:
You could create a standard error output UI which could trigger when an error is passed into it and display a formatted end-user-friendly error message; however, we do still want the actual error in the console for debugging purposes.